src/app/components/header/help-menu/help-menu.component.less

```diff
@@ -0,0 +1,3 @@
+.mat-icon{
```

Comment:
this style you need to place in app.component.less file

Reply:
If I add this style in the app. component, everywhere changing color. And I add this style only in this component, that style has only this button.
![image](https://user-images.githubusercontent.com/74968753/140648609-e9d28b5c-9a83-4bea-9003-7daa404e5bfe.png)
![image](https://user-images.githubusercontent.com/74968753/140648620-4f2ebde0-2bcd-427b-8bc9-c6a3b61d8804.png)